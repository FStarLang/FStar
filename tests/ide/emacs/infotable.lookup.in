{"query-id":"1","query":"push","args":{"kind":"lax","code":"module InfoTable\n","line":1,"column":0}}
{"query-id":"2","query":"push","args":{"kind":"lax","code":"\nlet x =\n  let aaa: int = 111 in\n  let bbb: string * int = \"AAA\", aaa in\n  let ccc: nat * (string * int) = (123 <: nat), bbb in\n  aaa, bbb, ccc\n","line":2,"column":0}}
{"query-id":"3","query":"lookup","args":{"context":"code","symbol":"ccc","requested-info":["type","documentation"],"location":{"filename":"<input>","line":7,"column":14}}}
{"query-id":"4","query":"lookup","args":{"context":"code","symbol":"aaa","requested-info":["type","documentation"],"location":{"filename":"<input>","line":5,"column":34}}}
{"query-id":"5","query":"lookup","args":{"context":"code","symbol":"bbb","requested-info":["type","documentation"],"location":{"filename":"<input>","line":6,"column":49}}}
