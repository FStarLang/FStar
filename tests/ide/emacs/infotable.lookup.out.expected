{"kind": "protocol-info", "rest": "[...]"}
{"kind": "response", "query-id": "1", "response": [], "status": "success"}
{"kind": "response", "query-id": "2", "response": [], "status": "success"}
{"kind": "response", "query-id": "3", "response": {"defined-at": null, "definition": null, "documentation": null, "kind": "symbol", "name": "ccc", "type": "nat * (string * int)"}, "status": "success"}
{"kind": "response", "query-id": "4", "response": {"defined-at": null, "definition": null, "documentation": null, "kind": "symbol", "name": "aaa", "type": "int"}, "status": "success"}
{"kind": "response", "query-id": "5", "response": {"defined-at": null, "definition": null, "documentation": null, "kind": "symbol", "name": "bbb", "type": "string * int"}, "status": "success"}
